### PR TITLE
Mailing List Validation

### PR DESCRIPTION
**Pull request by uavn**
https://github.com/mailgun/mailgun-php/issues/633

https://documentation.mailgun.com/en/latest/api-mailinglists.html#examples

### CODE REVIEW

**Review by DavidGarciaCat — APPROVED** [review]
Thanks

**Comment by DavidGarciaCat** [comment]
Hello @uavn 

Apparently, I cannot merge this branch, due to there are one or more conflicts.

Would you mind either rebase or merge from master, please, so we can get a new inspection from Travis and PrettyCI, and after that getting the chance to merge and tag?

Cheers,

**Comment by uavn** [comment]
> Hello @uavn
> 
> Apparently, I cannot merge this branch, due to there are one or more conflicts.
> 
> Would you mind either rebase or merge from master, please, so we can get a new inspection from Travis and PrettyCI, and after that getting the chance to merge and tag?
> 
> Cheers,

rebased

**Pull request merged**
